hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -155,9 +155,6 @@ class CommandInterface : public ReadWriteHandle<CommandInterface>
    * can't be copied in order to avoid simultaneous writes to
    * the same resource.
    */
-  CommandInterface(const CommandInterface & other) = delete;
-
-  CommandInterface(CommandInterface && other) = default;
```

Comment:
I don't think we should allow a copy constructor for a command interface. I don't see a use case for having a concurrent write to the same hardware instance.

Reply:
gone, I've created local handle types

Reply:
what do you mean with "gone"? The change is still present here.

Reply:
Updated the wrong branch, aplogies